infra/Pos/Slotting/Ntp.hs

```diff
@@ -65,12 +70,21 @@ makeLenses ''NtpSlottingState
 ----------------------------------------------------------------------------
 
 -- | Monad transformer which implements NTP-based solution for slotting.
-type NtpSlotting = Ether.ReaderT NtpSlottingVar
+-- Flag means whether to use real NTP servers or rely on local time.
+type NtpSlotting = Ether.ReaderT (Bool, NtpSlottingVar)
```

Comment:
@int-index 
>Can you make this Bool flag into a separate transformer?
I was considering such approach (and obliviously it's better than current implementation), but this task should be done quickly, and after dancing experience with stack transformers (working on csm-175), I rejected this approach.